app/views/brexit_checker/_results_criteria.html.erb

```diff
@@ -1,8 +1,13 @@
 <% if criteria.present? %>
   <div class="govuk-grid-column-one-third">
-  <% if local_assigns[:heading] %>
-    <h3 class="govuk-heading-m"><%= heading %></h3>
-  <% end %>
+    <% if local_assigns[:heading] %>
+      <%= render "govuk_publishing_components/components/heading", {
+      text: heading,
+      heading_level: 3,
+      font_size: "m",
+      margin_bottom: 4
+    } %>
```

Comment:
Super picky - the indentation here is off:
```suggestion
        text: heading,
        heading_level: 3,
        font_size: "m",
        margin_bottom: 4
      } %>
```

Reply:
Urgh. My beautifier totally missed this.